src/js/button.js

```diff
@@ -7,13 +12,13 @@
  * @class
  * @constructor
  */
-vjs.Button = vjs.Component.extend({
+var Button = Component.extend({
   /**
    * @constructor
    * @inheritDoc
    */
   init: function(player, options){
```

Comment:
I think that if we can make the migration plugin and have it come out in conjunction with the 5.0 release (what's the point otherwise?), then we could just pull the plug right now.